org/org.yaml

```diff
@@ -417,47 +417,47 @@ orgs:
         maintainers:
         - bobcatfish
         members:
-        - waveywaves
-        - sthaha
         - abayer
-        - shashwathi
-        - skaegi
-        - nader-ziada
-        - wlynch
-        - dlorenc
-        - dibbles
-        - pritidesai
-        - jerop
+        - afrittoli
+        - chuangw6
         - danielhelfand
-        - jlpettersson
+        - dibbles
         - dibyom
-        - piyush-garg
-        - chmouel
-        - psschwei
+        - dlorenc
         - hrishin
         - ImJasonH
-        - vdemeester
+        - jerop
+        - lbernick
         - mattmoor
+        - piyush-garg
+        - pritidesai
+        - psschwei
+        - skaegi
+        - vdemeester
+        - waveywaves
         - withlin
-        - afrittoli
-        - pierretasci
-        - lbernick
+        - wlynch
+        - XinruZhang
         - ywluogg
         - Yongxuanzhang
-        - chuangw6
-        - XinruZhang
         # alumni:
         # AlanGreene
         # barthy1
+        # chmouel
         # dwnusbaum
         # EliZucker
         # FogDong
         # GregDritschler
         # houshengbo
+        # jlpettersson
+        # nader-ziada
         # othomann
         # Peaorl
+        # pierretasci
```

Comment:
😢 I wish I could contribute more

Reply:
Thanks Pierre! We appreciate all kinds of contributions (bug reports, PRs, answering questions, etc). If you anticipate doing more reviews I'm happy to update this PR to keep you in core.collaborators.